contracts/test/ocr2/aggregator.test.ts

```diff
@@ -17,44 +17,33 @@ const CHUNK_SIZE = 31
 // Observers - max 31 oracles or 31 bytes
 const OBSERVERS_MAX = 31
 const OBSERVERS_HEX = '0x00010203000000000000000000000000000000000000000000000000000000'
-const INT128_MIN = BigInt(-2) ** BigInt(128 - 1)
-const INT128_MAX = BigInt(2) ** BigInt(128 - 1) - BigInt(1)
+const UINT128_MAX = BigInt(2) ** BigInt(128) - BigInt(1)
 
-function encodeBytes(data: Uint8Array): string[] {
-  let felts = []
+function packUint8Array(data: Uint8Array | Buffer): bigint {
```

Comment:
These util functions seem duplicated from here: [packages-ts/starknet-gauntlet-ocr2/src/lib/encoding.ts](https://github.com/smartcontractkit/chainlink-starknet/blob/0fadd26e2e2a993197a6e1cbaf515399ea85be29/packages-ts/starknet-gauntlet-ocr2/src/lib/encoding.ts)

Consider finding a shared place for those - probably `packages-ts/starknet` would make sense.